### PR TITLE
VAVSA-8869: Expand Benefits Search Radius for "All VA Benefits" searches

### LINKED ISSUE #8869: Facility Locator expected behavior for VBA "all services" searches
## Issue Description
The Facility Locator is designed to return the closest locations which map to the search parameters. However, there may be a disconnect in the "All services" default, particularly for "VA benefit" searches. 

This is well illustrated by this [example search](https://www.va.gov/find-locations/?page=1&address=22903&facilityType=benefits&serviceType&latitude=38.03&longitude=-78.5&radius=19&bounds%5B%5D=-79.25&bounds%5B%5D=37.28&bounds%5B%5D=-77.75&bounds%5B%5D=38.78&context=Charlottesville%2C%20Virginia%2022903%2C%20United%20States)
1. Enter 22903 as zip code
2. Choose VE benefits
3. Choose All VA benefit services
4. Observe single result is returned: [VetSuccess on Campus at Liberty University](https://www.va.gov/find-locations/facility/vba_314k). This location is on a campus and offers a single service: Education and career counseling.

Returning a location offering a single, particular service is not consistent with an "all services" parameter. Should "all services" map to Benefit region offices which actually offer "all services"? How do we best support Veterans, particularly if no benefit region offices exist within the searched geography?

---
## Tasks
- [ ] Determine ideal behavior
- [ ] Determine how best to support Veterans in a variety of possible scenarios

## Acceptance Criteria
- [ ] Searching for "VA Benefits" -> "All VA Benefit Services" should show the nearest Regional Office as well as any other facilities that may fall within that range.
---

### PR DESCRIPTION
**Pull request by maxx1128**
## Description

Closes [department-of-veterans-affairs/va.gov-team#0000](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/8869)

## Original issue(s)

[department-of-veterans-affairs/va.gov-team#0000](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/8869)

## Testing done

Visual

## Screenshots

Screen Shot 2022-04-29 at 11 28 03 AM

## Testing Setup Instructions

- [ ] In the local instance of `vets-website` repo, change `src/applications/facility-locator/config.js` replace lines 23-28 with:

```javascript
const railsEngineApi = {
 baseUrl: `https://api.va.gov/facilities_api/v1`,
 url: `https://api.va.gov/facilities_api/v1/va`,
 ccUrl: `https://api.va.gov/facilities_api/v1/ccp`,
 settings: apiSettings,
};
```

- [ ] Disable CORS in chrome - for reference: https://alfilatov.com/posts/run-chrome-without-cors/
- [ ] in `vets-website` repo, run `yarn watch`
- [ ] Once the above step has compiled, go to `http://localhost:3001/find-locations/`

## Acceptance criteria

- [ ] Enter "22903" for the zip code, select "VA benefits" for the facility type & "All VA benefit services" for the service type & press Search. Confirm that multiple facilities are listed due to an expanded bounding box, and that "Veteran Readiness and Employment Office at Veterans Affairs Acquisition Academy" is one of them.
- [ ] Change the service type to "vocational rehabilitation and employment help," and confirm that "Veteran Readiness and Employment Office at Veterans Affairs Acquisition Academy" is no longer listed. This will confirm that the bounding box has shrunk now that "All VA Benefits" is no longer being searched.

### CODE REVIEW

**Review by ethanteague — APPROVED** [review]
Nice work, @maxx1128

File: src/applications/facility-locator/constants/index.js [hunk 1]
```diff
@@ -59,6 +59,7 @@ export const LOCATION_OPTIONS = [
  * Defines the ± change in bounding box size for the map when changing zoom
  */
 export const BOUNDING_RADIUS = 0.75;
+export const EXPANDED_BOUNDING_RADIUS = 1.4;
```
Comment (swirtSJW) [inline src/applications/facility-locator/constants/index.js:62]: Curious:  How did 1.4 get decided upon?  Was that just the minimum bump that worked?

**Comment by swirtSJW** [comment]
Overall this looks great.   Nice work on the  attention to detail on the testing steps and AC checks.  I do want to have @ryguyk take a look as he often catches things I don't.

File: src/applications/facility-locator/containers/FacilitiesMap.jsx [hunk 2]
```diff
@@ -70,19 +70,26 @@ const FacilitiesMap = props => {
       return;
     }
     const { location } = props;
+    const {
+      query: { address, context, facilityType, serviceType },
+    } = location;
+    const expandedRadius = facilityType === 'benefits' && !serviceType;
 
     if (!isEmpty(location.query)) {
```
Comment (ryguyk) [inline src/applications/facility-locator/containers/FacilitiesMap.jsx:78]: If it is, indeed, possible that `location.query` is empty, then line 74 above might also produce an error. Object destructuring can be a very good tool, but it needs some caution.

File: src/applications/facility-locator/constants/index.js (hunk 1 above)
Comment (ethanteague) [reply src/applications/facility-locator/constants/index.js:62]: Yep.

**Review by ryguyk — COMMENTED** [review]
Looks like a lot of this is a move to use object destructuring. I'm curious if this makes things more clear in all cases. I don't have a strong overarching opinion.  However, see one specific comment about how this might introduce a bug.

File: src/applications/facility-locator/containers/FacilitiesMap.jsx (hunk 2 above)
Comment (maxx1128) [reply src/applications/facility-locator/containers/FacilitiesMap.jsx:78]: This is a good catch with the destructuring I didn't notice before. I also saw that the `expandedRadius` declaration should be inside the `location.query.address` conditional - it doesn't make sense to create the variable before the code even knows if it's going to be used or not.

**Review by ryguyk — APPROVED** [review]
Nice fix!

**Pull request merged**